### PR TITLE
Add test case in search_indexer.go

### PR DESCRIPTION
**Pull request by liuzhengbaihe**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-PACKAGES = $(shell find ./src/ -type d -not -path '*/\.*')
+PACKAGES = $(shell go list ./src/...)
```
Comment (vitan) [inline Makefile]: 👍

**Comment by xiaods** [comment]
this is outdate ?

**Comment by xiaods** [comment]
ping @liuzhengbaihe

**Comment by liuzhengbaihe** [comment]
@xiaods, not outdate. unittesting failed, I'm fixing.

**Comment by liuzhengbaihe** [comment]
@xiaods , @vitan , this pr is fixed. please verify.

**Comment by vitan** [comment]
LGTM

We will apply the mock server to other unittest cases

**Pull request merged**